assets/aws/Makefile

```diff
@@ -23,17 +23,14 @@ TELEPORT_UID ?= 1007
 INSTANCE_TYPE ?= t2.medium
 
 # Use comma-separated values without spaces for multiple regions
-# For now, limit AMI to regions with DynamoDB encryption at rest enabled
-# https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/EncryptionAtRest.html
-# https://aws.amazon.com/about-aws/whats-new/2018/04/amazon-dynamodb-encryption-at-rest-now-available-in-additonal-regions/
-DESTINATION_REGIONS ?= us-east-1,us-east-2,us-west-1,us-west-2,ap-south-1,ap-northeast-2,ap-southeast-1,ap-southeast-2,ap-northeast-1,ca-central-1,eu-central-1,eu-west-1,eu-west-2,sa-east-1
+# For now, limit AMI publishing to non opt-in regions
+# https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
+# You can get this with $(AWS_DEFAULT_REGION=us-west-2 aws ec2 describe-regions | jq -r '.Regions | map(.RegionName) | join(",")')
+DESTINATION_REGIONS ?= eu-north-1,ap-south-1,eu-west-3,eu-west-2,eu-west-1,ap-northeast-3,ap-northeast-2,ap-northeast-1,sa-east-1,ca-central-1,ap-southeast-1,ap-southeast-2,eu-central-1,us-east-1,us-east-2,us-west-1,us-west-2
 
 # Generate timestamp for builds
 BUILD_TIMESTAMP := $(shell TZ=UTC /bin/date "+%Y%m%d-%H%M%S%Z")
 
-# S3 Bucket ID
-S3_BUCKET_ID ?=
```

Comment:
remove the mention of `S3_BUCKET_ID` from https://github.com/gravitational/teleport/blob/master/assets/aws/README.md#L44